engines/libaio.c

```diff
@@ -83,6 +89,15 @@ static struct fio_option options[] = {
 		.help	= "Your platform does not support I/O priority classes",
 	},
 #endif
+	{
```

Comment:
Should this be down as `FIO_OPT_UNSUPPORTED` if `CONFIG_LIBAIO_RW_FLAGS` isn't defined? See https://github.com/axboe/fio/blob/2f6cbfdcf0ce3b53e83713e3a6b5a0bb6a76a1e9/engines/io_uring.c#L130 for an example.

Reply:
If CONFIG_LIBAIO_RW_FLAGS isn' t defined there is hack above: #define aio_rw_flags __pad2
So it this works for old libaio too. Older kernels return EINVAL if it's non-zero. 4.14+ use it for flags.